Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/command-reference/remote/add.md

> See `dvc remote modify` for a full list of SSH parameters.

⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.

> Do check whether you are able to connect both ways with tools like `ssh` and

Reviewer: ```suggestion
Check that you can connect both ways with tools like `ssh` and
```
